src/main.rs

```diff
@@ -445,6 +467,84 @@ fn analog_write(spi_drv: &mut SpiDrv, uart: &mut EnabledUart, pin: u8, value: u8
     spi_drv.esp_deselect();
 }
 
+fn wifi_set_passphrase(spi_drv: &mut SpiDrv, uart: &mut EnabledUart, mut ssid: String<STR_LEN>, mut passphrase: String<STR_LEN>) -> bool {
+    spi_drv.wait_for_esp_select();
+
+    // Send Command
```

Comment:
```suggestion
```

Reply:
Resolved in [following commit](https://github.com/Jim-Hodapp-Coaching/esp32-pico-wifi/pull/4/commits/c10bb540c941bdb44dbff93ce28af3736f1ca7b1)